Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```diff
@@ -453,14 +455,15 @@
   - type: Clothing
     sprite: Clothing/Mask/fox.rsi
   - type: BreathMask
+  - type: IngestionBlocker
   - type: IdentityBlocker
   - type: Tag
     tags:
     - IPCMaskWearable
   - type: HideLayerClothing
     slots:
     - Snout
-
+ hideOnToggle: true
```

Comment:
You've deleted and re added this. Make sure it's formatted correctly and indented to the correct component above.